src/test/java/guitests/KeyBindingsGuiTest.java

```diff
@@ -15,13 +15,13 @@
 
     private AddressBook initialData = generateInitialData();
 
-    private AddressBook generateInitialData() {//TODO: create a better set of sample data
+    private AddressBook generateInitialData() {
         AddressBook ab = new AddressBook();
-        ab.addPerson(new Person("Person1", "Lastname1", 1));
-        ab.addPerson(new Person("Person2", "Lastname2", 2));
-        ab.addPerson(new Person("Person3", "Lastname3", 3));
-        ab.addPerson(new Person("Person4", "Lastname4", 4));
-        ab.addPerson(new Person("Person5", "Lastname5", 5));
+        ab.addPerson(new Person("John", "Wilson", 1));
+        ab.addPerson(new Person("John", "Lennon", 2));
+        ab.addPerson(new Person("Allan", "Turing", 3));
+        ab.addPerson(new Person("Obama", "Micheal", 4));
+        ab.addPerson(new Person("Pedo", "Lee", 5));
         return ab;
     }
```

Comment:
Use the same set of `typical data` (used in PersonEditGuitTest) here as well? Also note the naming convention used there. It's not totally random.